### ISSUE TITLE
ter-indent: Does not report missing indents in chains and ternaries

### ISSUE THREAD

**Issue by woutervanvliet** [buzinas/tslint-eslint-rules#373]
I'm trying to configure `ter-indent` to behave as similar to the eslint `indent` rule I've been using for a while, and for the most part this seems to function alright. However in some cases it doesn't report missing indents - where eslint does.

Specifically, I've identified the problem in the following cases:
```ts
// Here I would like to force an indent before ? and before :
export const theValue = document.location.search === ""
? "empty-search"
: "not-empty-search";

// here I'd like to force an indent before .
export const zero = ["one"]
.map((hi) => hi.toUpperCase());
```

I've setup a minimal repo to demonstrate the issue: https://github.com/woutervanvliet/ter-indent-problem

Would this be something that can be fixed? Or is there some configuration property I'm missing that can enable the desired behaviour in my project?

Thanks!
